aws/eks-anywhere
# Add vSphere user and ssh validations to vspheremachineconfig webhook

*Issue #, if available:*

*Description of changes:*

*Testing (if applicable):*
- Unit tests
- Manual testing
- Create a management cluster:
  - Create and update workload cluster with API .
      - No users
      - User with empty name
      - User with empty SSHAuthorizedKeys 
      - User specified with name and valid sshAuthorizationKey (succeeds)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
